checker/tests/nullness-extra/issue265/Makefile

```diff
@@ -1,8 +1,10 @@
 .PHONY: all
 
-all:
+all: clean
 	mkdir -p bin
 	$(JAVAC) -processor org.checkerframework.checker.nullness.NullnessChecker *.java -d bin/
 	$(JAVAC) -processor org.checkerframework.checker.nullness.NullnessChecker Delta.java -cp bin/ -d bin/
-	rm bin/*.class
+
+clean:
 	rmdir bin
+	rm bin/*.class
```

Comment:
If there are still classes in `bin`, the `rmdir` above will fail.

Reply:
Reserved the order.